src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -93,9 +93,9 @@ const SidebarSidecarLayoutContent = ({
         >
           <SidebarContent />
         </motion.div>
-        <div className={s.mainArea}>
-          <div className={s.main}>
-            <main id="main">
+        <div className={s.mainAreaAndFooter}>
+          <div className={s.mainArea}>
+            <main id="main" className={s.main}>
```

Comment:
I've avoided creating an extra wrapper div by moving the `.main` className to the `<main>` element.